src/register/data/constants.js

```diff
@@ -39,137 +39,4 @@ export const EDUCATION_LEVELS = [
 
 export const GENDER_OPTIONS = ['', 'f', 'm', 'o'];
```

Comment:
`YEAR_OF_BIRTH_OPTIONS`, `EDUCATION_LEVELS` and `GENDER_OPTIONS` are not being used. Do we need these?

Reply:
Yep! this was in my todo to confirm if we need these or not. I will remove them. I have a question, why they are not being used now?

Reply:
We are getting the data from backend. (dynamic optional registration fields)